kinesis.tf

```diff
@@ -0,0 +1,27 @@
+module "kinesis_label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.1"
+  namespace  = "${var.namespace}"
+  name       = "${var.name}"
+  stage      = "${var.stage}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${compact(concat(var.attributes, list("kinesis")))}"
+  tags       = "${var.tags}"
+}
+
+resource "aws_kinesis_stream" "default" {
+  name                = "${module.kinesis_label.id}"
+  shard_count         = "${var.shard_count}"
+  retention_period    = "${var.retention_period}"
+  shard_level_metrics = "${var.shard_level_metrics}"
+  encryption_type     = "${var.encryption_type}"
+  kms_key_id          = "${var.encryption_type}"
+  tags                = "${module.kinesis_label.tags}"
+}
+
+resource "aws_cloudwatch_log_subscription_filter" "default" {
+  name            = "${aws_cloudwatch_log_group.default.name}"
```

Comment:
`aws_cloudwatch_log_group` name (`/aws/lambda/example_lambda_name`) may be improper for resource name.
https://www.terraform.io/docs/providers/aws/r/cloudwatch_log_subscription_filter.html

We should use `terraform-null-label` for resource names.

Reply:
>aws_cloudwatch_log_group name (/aws/lambda/example_lambda_name) may be improper for resource name.

it doesn't matter